tsdb/query_executor.go

```diff
@@ -748,14 +751,6 @@ func (q *QueryExecutor) writeInto(row *models.Row, selectstmt *influxql.SelectSt
 	if err != nil {
 		return err
 	}
-	for _, p := range points {
```

Comment:
Because it drops the entirety of the query if it finds a nil. After I pulled it into the into.go file, it will only drop the relevant fields.